tests/xrayignorerule_test.go

```diff
@@ -0,0 +1,130 @@
+package tests
+
+import (
+	"fmt"
+	"testing"
+	"time"
+
+	"github.com/jfrog/jfrog-client-go/xray/services/utils"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestXrayIgnoreRule(t *testing.T) {
+	initXrayTest(t)
+	t.Run("createCveIgnoreRule", createCveIgnoreRule)
+	t.Run("createVulnerabilitesAndLicensesIgnoreRule", createVulnerabilitesAndLicensesIgnoreRule)
+	t.Run("createIgnoreRuleOnWatch", createIgnoreRuleOnWatch)
+}
+
+func deleteIgnoreRule(t *testing.T, ignoreRuleId string) {
+	err := testsXrayIgnoreRuleService.Delete(ignoreRuleId)
+	assert.NoError(t, err)
+}
+
+func createCveIgnoreRule(t *testing.T) {
+	var ignoreRuleId string
+	defer func() {
+		if ignoreRuleId != "" {
+			deleteIgnoreRule(t, ignoreRuleId)
+		}
+	}()
+
+	component := utils.IgnoreFilterNameVersion{
+		Name:    "gav://org.postgresql:postgresql",
+		Version: "42.2.3.jre7",
+	}
+	components := []utils.IgnoreFilterNameVersion{component}
+
+	cve := []string{"CVE-2022-31197"}
+	ignoreRuleFilter := utils.IgnoreFilters{
+		CVEs:       cve,
+		Components: components,
+	}
+
+	ignoreRuleId = createIgnoreRule(t, ignoreRuleFilter)
```

Comment:
This comment applies to then other tests that create ignore rules as well - 
Shouldn't we check that the `ignoreRuleId` isn't empty?

Reply:
I had the check at defer function and changed it to be in the function body with assertion.
If the ignoreRuleId is empty I would expect an error to occur earlier at the API phase but I don't want to ignore it in the test as it is an error.